src/scripts/ci_build.py

```diff
@@ -482,10 +481,16 @@ def main(args=None):
             print('Error unknown compiler %s' % (options.cc))
             return 1
 
-    if options.compiler_cache is None and options.cc != 'msvc':
-        # Autodetect ccache
-        if have_prog('ccache'):
+    if options.compiler_cache is None:
+        # Autodetect compiler cache
+        if have_prog('sccache'):
+            print('Found \'sccache\' installed, will use it...')
```

Comment:
Instead of these two `print`, have one `print` at the end that prints `options.compiler_ccache` if it is not `None`

Reply:
Also you can use "" for the outer quotes and then don't have to quote the `'`